Actions/DetermineDeploymentEnvironments/DetermineDeploymentEnvironments.ps1

```diff
@@ -173,7 +173,14 @@ else {
             foreach($key in $keys) {
                 if ($deploymentSettings.ContainsKey($key)) {
                     if ($null -ne $deploymentSettings."$key" -and $null -ne $deployTo."$key" -and $deploymentSettings."$key".GetType().Name -ne $deployTo."$key".GetType().Name) {
-                        Write-Host "::WARNING::The property $key in $settingsName is expected to be of type $($deploymentSettings."$key".GetType().Name)"
+                        if ($key -eq "runs-on" -and $deployTo."$key" -is [Object[]]) {
+                            # Support setting runs-on as an array in settings to not break old settings
+                            # See https://github.com/microsoft/AL-Go/issues/1182
+                            $deployTo."$key" = $deployTo."$key" -join ','
```

Comment:
What about SETTINGS.md?

Reply:
In settings.md it isn't described as a specific type and people tend to use a string (like the original runs-on)